src/jolt/vm/pc.rs

```diff
@@ -0,0 +1,549 @@
+use std::marker::PhantomData;
+
+use ark_ec::CurveGroup;
+use ark_ff::{PrimeField, Field};
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+use ark_std::{Zero, One};
+use merlin::Transcript;
+
+use crate::{
+  lasso::{fingerprint_strategy::{FingerprintStrategy, MemBatchInfo}, gp_evals::GPEvals},
+  poly::{dense_mlpoly::{DensePolynomial, PolyCommitmentGens}, identity_poly::IdentityPolynomial},
+  subprotocols::{
+    combined_table_proof::{CombinedTableCommitment, CombinedTableEvalProof},
+    grand_product::BGPCInterpretable,
+  },
+  utils::{self, errors::ProofVerifyError, math::Math, transcript::ProofTranscript},
+};
+
+pub struct PCPolys<F: PrimeField> {
+  a_ops: DensePolynomial<F>,
+  v_ops: DensePolynomial<F>,
+
+  v_mem: DensePolynomial<F>,
+
+  t_read: DensePolynomial<F>,
+  t_final: DensePolynomial<F>,
+
+  /// Starting address for a_ops
+  address_offset: usize,
```

Comment:
imo just delete the `address_offset` until we're sure we need to be handling it here

Reply:
Done.